config/Hipcheck.kdl

```diff
@@ -28,7 +28,7 @@ analyze {
     }
 
     category "attacks" {
-        analysis "mitre/typo" policy="(eq 0 (count $))" {
+        analysis "mitre/typo" policy="(lte (count (filter (eq #t) $)) 0)" {
```

Comment:
From the typo plugin, the default is `(lte (count (filter (eq #t) $)) {}))`. I would update this policy to match, or simply provide a `count_threshold` config field